contrib/platform/src/com/sun/jna/platform/win32/GDI32Util.java

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2015 Michael Freeman, All Rights Reserved
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.  
+ */
+package com.sun.jna.platform.win32;
+
+import java.awt.Rectangle;
+import java.awt.image.BufferedImage;
+
+import com.sun.jna.Memory;
+import com.sun.jna.platform.win32.WinDef.HBITMAP;
+import com.sun.jna.platform.win32.WinDef.HDC;
+import com.sun.jna.platform.win32.WinDef.HWND;
+import com.sun.jna.platform.win32.WinDef.RECT;
+import com.sun.jna.platform.win32.WinGDI.BITMAPINFO;
+import com.sun.jna.platform.win32.WinNT.HANDLE;
+
+/**
+ * GDI32 utility API.
+ * 
+ * @author mlfreeman[at]gmail.com
+ */
+public class GDI32Util {
+	/**
+	 * Takes a screenshot of the given window
+	 * 
+	 * @param target
+	 *            The window to target
+	 * @param windowWidth
+	 *            the width of the window
+	 * @param windowHeight
+	 *            the height of the window
+	 * @return the window captured as a screenshot
+	 */
+	public static BufferedImage getScreenshot(HWND target) {

```

Comment:
There's no error handling here. Each one of those Win32 calls can fail, result needs to be checked and errors raised.